assets/section-main-product.css

```diff
@@ -11,13 +11,18 @@
   margin: 0 auto;
 }
 
+@media screen and (min-width: 750px) {
+  .product--reverse {
+    flex-direction: row-reverse;
+  }
+}
+
```

Comment:
I guess it'd be good to loop in @svinkle to confirm it's ok. 
Context: 
>We're adding a setting to choose whether you want your product media showing first on desktop (so to the left like it's been so far) or second (to the right).
When using `flex-direction: row-reverse;` it means that when tabbing through you're going to land on the images/medias first, whatever the order chosen is. 
[Here is Olivia's video that shows the tabbing order](https://screenshot.click/06-07-gkeh6-sr3zh.mp4)

A word around would be to look into using grid and assigning the two different blocks to a specific column. 🤔

Reply:
Thanks for mentioning Scott.

As we wait for his feedback, do you happen to have more context on why the "Skip to product information" was originally added? See the video in my PR description > [Accessibility](https://github.com/Shopify/dawn/pull/1720#:~:text=keep%20things%20simple.-,Accessibility,-For%20accessibility%20) but I couldn't find any prior issues etc.

Reply:
K so it seems that ideally we wouldn't rely on flex's `row-reverse` and keep the natural tab order. 

So I'm guessing we could capture the content of one of the "column" into a `capture` and output it either before or after the other column based on the setting. 

```html
{% capture media_column %}
  <div class="grid__item product__media-wrapper">
    ...
```

Reply:
Thanks Ludo. I'm using the `capture` method you suggested (running it locally) and it's working great. Since this is a "Desktop only" enhancement, do you have offhand suggestions on how to maintain the "image on top" / have the media wrapper "on top" on mobile? I could be missing something that's probably really simple 🤔 

https://user-images.githubusercontent.com/28404165/173429965-a77b42db-53a5-4967-bea4-d82e4ea95188.mp4

Reply:
Hmm then I guess it wouldn't be with a capture 🤔 But duplicate code. Always keep the original one and apply a hiding class on desktop if the medias aren't set to come up first/on the left. 

And add `small-hide` to the block containing the medias when set to be on the right. That's going to make it a big file 😬 
I wonder if this could create some issues in terms of IDs and behaviours (showing the image variant when it's being selected, etc).

Reply:
Thanks Ludo. I've added some more changes based on your suggestions here. Let me know your thoughts / feedback.